pulseapi/settings.py

```diff
@@ -157,3 +165,12 @@
 CORS_ORIGIN_WHITELIST = os.getenv('CORS_ORIGIN_WHITELIST', 'localhost:3000,localhost:8000,localhost:8080,test.example.com:8000,test.example.com:8080,pulse-react.herokuapp.com').split(',')
 
 CORS_ORIGIN_REGEX_WHITELIST = []
+CSRF_COOKIE_SECURE = True
+CSRF_COOKIE_HTTPONLY = True
+SECURE_BROWSER_XSS_FILTER = True
+SESSION_COOKIE_SECURE = True
+X_FRAME_OPTIONS = "DENY"
+SECURE_CONTENT_TYPE_NOSNIFF = True
+SECURE_SSL_REDIRECT = env('SECURE_SSL_REDIRECT')
+SECURE_HSTS_SECONDS = 63072000
```

Comment:
a small nit, but I like being explicit when I'm setting timeouts in seconds. `60 * 60 * 24 * 365 * 2` is a one-time evaluation that makes it much more obvious this code is setting a timeout of two years, measured in seconds.

Alternatively:
```
SECURE_HSTS_SECONDS = datetime.timedelta(weeks=104).total_seconds()
```